client/my-sites/subscribers/subscriber-profile.tsx

```diff
@@ -0,0 +1,19 @@
+export function SubscriberProfile( {
+	avatar,
+	displayName,
+	email,
+}: {
+	avatar: string;
+	displayName: string;
+	email: string;
+} ) {
+	return (
+		<div className="user-wrapper">
```

Comment:
Same here for the className conventions.

Reply:
I'll change them